src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html

```diff
@@ -0,0 +1,91 @@
+                  <input class="govuk-radios__input" id="completed" formName="hearingResult" name="hearingResult"
+                    type="radio" value="completed" data-aria-controls="conditional-completed" (change)="onHearingResult(hearingResultEnum.COMPLETED)">
+                  <label class="govuk-label govuk-radios__label" for="completed">
+                    Heard, order or decision
+                  </label>
+                </div>
+                <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.COMPLETED">
+                  <div class="govuk-form-group">
+                    <label class="govuk-label" id="dropdown-label" for="completed-reason">Hearing outcome</label>
+                    <select formControlName="completedReason" class="govuk-select" id="completed-reason" name="completedReason">
+                      <option value="">Please select</option>
+                      <option *ngFor="let reason of completeHearingActualReasons$ | async" [ngValue]="reason.key">{{reason.value_en}}</option>
+                    </select>
+                  </div>
+                </div>
+                <div class="govuk-radios__item">
+                  <input class="govuk-radios__input" id="adjourned" formName="hearingResult" name="hearingResult"
+                    type="radio" value="adjourned" data-aria-controls="conditional-adjourned" (change)="onHearingResult(hearingResultEnum.ADJOURNED)">
+                  <label class="govuk-label govuk-radios__label" for="adjourned">
+                    Part-heard, adjourned or postponed
+                  </label>
+                </div>
+                <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.ADJOURNED">
+                  <div class="govuk-form-group">
+                    <label class="govuk-label" id="dropdown-label" for="adjourned-reason">Select a reason for this result</label>
+                    <select formControlName="adjournedReason" class="govuk-select" id="adjourned-reason" name="adjournedReason">
+                      <option value="" selected>Please select</option>
+                      <option *ngFor="let reason of adjournHearingActualReasons$ | async" [ngValue]="reason.key">{{reason.value_en}}</option>
+                    </select>
+                  </div>
+                </div>
+                <div class="govuk-radios__item">
+                  <input class="govuk-radios__input" id="cancelled" formName="hearingResult" name="hearingResult"
+                    type="radio" value="cancelled" data-aria-controls="conditional-cancelled" (change)="onHearingResult(hearingResultEnum.CANCELLED)">
+                  <label class="govuk-label govuk-radios__label" for="cancelled">
+                    Withdrawn, cancelled or vacated
+                  </label>
+                </div>
+                <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.CANCELLED">
```

Comment:
could this change to NgSwitchCase?

Reply:
@rayliangatos - I think NgSwitchCase is not possible in this scenario as each dropdown is a child of radio button and we have to show all the radio buttons by default and show/hide dropdowns based on radio button selection.